src/platform_impl/android/mod.rs

```diff
@@ -1021,32 +1020,11 @@ impl MonitorHandle {
         unreachable!()
     }
 
-    pub fn current_video_mode(&self) -> Option<VideoModeHandle> {
+    pub fn current_video_mode(&self) -> Option<VideoMode> {
         unreachable!()
     }
 
-    pub fn video_modes(&self) -> std::iter::Empty<VideoModeHandle> {
-        unreachable!()
-    }
-}
-
-#[derive(Clone, Debug, Eq, Hash, PartialEq)]
-pub struct VideoModeHandle;
-
-impl VideoModeHandle {
-    pub fn size(&self) -> PhysicalSize<u32> {
-        unreachable!()
-    }
-
-    pub fn bit_depth(&self) -> Option<NonZeroU16> {
-        unreachable!()
-    }
-
-    pub fn refresh_rate_millihertz(&self) -> Option<NonZeroU32> {
-        unreachable!()
-    }
-
-    pub fn monitor(&self) -> MonitorHandle {
+    pub fn video_modes(&self) -> std::iter::Empty<VideoMode> {
         unreachable!()
```

Comment:
On `orbital` you implemented this to return `std::iter::empty()`, I'm not sure what to do here. There, the type is `impl Iterator` and not `std::iter::Empty` too.

Reply:
You can not have `unreachable!()` with `impl Iterator`.

Reply:
Hence the question is: should Android return `std::iter::empty()` with `std::iter::Empty<>` type?